CHANGELOG.md

```diff
@@ -65,6 +65,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Removed
 - `balanceAndSignTxE`, `balanceAndSignTx`, `balanceAndSignTxs`, `balanceTxWithAddress`, `balanceTxsWithAddress`, `withBalancedAndSignedTx` and `withBalancedAndSignedTxs` from `Contract.Transaction` ([#1053](https://github.com/Plutonomicon/cardano-transaction-lib/pull/1053))
+- `ScriptOutput` is deprecated and therefore not exported by `Contract.Address` anymore. also, `Contract.Transaction` doesn't export `scriptOutputToTransactionOutput` anymore ([#1153](https://github.com/Plutonomicon/cardano-transaction-lib/pull/1153)).
```

Comment:
Removed, not deprecated.

Reply:
Also: link to the issue and not to the PR please

Reply:
Until now I always linked to the PR and not to the Issue (as I saw that most changes in the CHANGELOG are). should the default be linking to the issue?

Reply:
They are almost always interlinked, so it's not critical, but the issue (ideally) should explain the motivation for the change better.